<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DealDamageDialog</class>
 <widget class="QWidget" name="DealDamageDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>215</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Deal Damage</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Target</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radio_enemy">
        <property name="text">
         <string>Enemy</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radio_actor">
        <property name="text">
         <string>Actor</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="combo_enemy">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>Entire Troop</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1.</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2.</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3.</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4.</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5.</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6.</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7.</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8.</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="combo_actor">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>Entire Party</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Actor No. 1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Actor No. 2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Actor No. 3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Actor No. 4</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="button_ok">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Operand</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radio_constant">
        <property name="text">
         <string>Constant</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="radio_variable">
        <property name="text">
         <string>Variable</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spin_constant">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>-999999</number>
        </property>
        <property name="maximum">
         <number>999999</number>
        </property>
        <property name="value">
         <number>999999</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="combo_variable">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>radio_enemy</sender>
   <signal>toggled(bool)</signal>
   <receiver>combo_enemy</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_actor</sender>
   <signal>toggled(bool)</signal>
   <receiver>combo_actor</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_constant</sender>
   <signal>toggled(bool)</signal>
   <receiver>spin_constant</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_variable</sender>
   <signal>toggled(bool)</signal>
   <receiver>combo_variable</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
